components/upload-photo/domain/s3.service.ts

```diff
@@ -0,0 +1,30 @@
+import { S3Client, PutObjectCommand } from "@aws-sdk/client-s3";
+
+const s3 = new S3Client({
```

Comment:
this approach an be risky to instantiate multiple instances of the same s3 client. Think about turning it to the singleton - one source, or even consider using class